games-emulation/virtualjaguar/virtualjaguar-2.1.2.ebuild

```diff
@@ -3,7 +3,7 @@
 # $Header: /var/cvsroot/gentoo-x86/games-emulation/virtualjaguar/virtualjaguar-2.1.2.ebuild,v 1.1 2014/10/15 03:50:52 mr_bones_ Exp $
 
 EAPI=5
-inherit eutils qt4-r2 games
+inherit qt4-r2 games
```

Comment:
actually "domenu" use it now, but eutils eclass automatically inherited with games eclass... and we actually do it twice in a row.